### PR TITLE
Introduce tooltip fields to document layer specification

### PR DESCRIPTION
**Pull request by VijayanB**
### Description
Added tooltip as field of type array of string.
This will save tooltip field selected by users.

Also added show tooltips or not. This will be useful if user want to temporarily disable tooltip

Signed-off-by: Vijayan Balasubramanian <balasvij@amazon.com>

https://user-images.githubusercontent.com/11067894/207951565-766d998c-8ea4-4923-a837-5209576cac33.mov

### Issues Resolved

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by VijayanB** [comment]
This PR will not update maplibre instance yet. This will be done in succession PR.

File: maps_dashboards/public/components/layer_config/documents_config/document_layer_source.tsx [hunk 1]
```diff
@@ -129,79 +187,130 @@ export const DocumentLayerSource = ({
...
+            </EuiFlexGrid>
+          </EuiForm>
+        </EuiCollapsibleNavGroup>
+      </EuiPanel>
+      <EuiSpacer size="m" />
+      <EuiPanel paddingSize="s">
+        <EuiCollapsibleNavGroup
+          title="Tool Tips"
+          titleSize="xxs"
+          isCollapsible={true}
```
Comment (junqiu-lei) [inline maps_dashboards/public/components/layer_config/documents_config/document_layer_source.tsx:283]: is it by designed to be true by default?

File: maps_dashboards/public/model/mapLayerType.ts [hunk 2]
```diff
@@ -36,6 +36,8 @@ export type DocumentLayerSpecification = {
     geoFieldType: 'geo_point' | 'geo_shape';
     geoFieldName: string;
     documentRequestNumber: number;
+    showTooltips: boolean;
```
Comment (junqiu-lei) [inline maps_dashboards/public/model/mapLayerType.ts:39]: We might also need to update the initial config here: 
https://github.com/opensearch-project/dashboards-maps/blob/feature/new-maps/maps_dashboards/public/utils/getIntialLayerConfig.ts

Comment (VijayanB) [reply maps_dashboards/public/model/mapLayerType.ts:39]: updated with default values

**Review by junqiu-lei — APPROVED** [review]

**Review by junqiu-lei — APPROVED** [review]

**Review by martin-gaievski — APPROVED** [review]

**Pull request merged**